borrowings/views.py

```diff
@@ -57,3 +58,20 @@ def get_serializer_class(self):
 
     def perform_create(self, serializer):
         serializer.save(user=self.request.user)
+
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "is_staff",
+                type={"type": "list", "items": {"type": "numbers"}},
+                description="Filter by users(ex. ?user_id=1)"
+            ),
+            OpenApiParameter(
+                "borrowed",
+                type={"type": "list", "items": {"type": "numbers"}},
+                description="Filter by status of book is borrowed(ex. ?is_active=True)"
+            ),
+        ]
+    )
```

Comment:
These parameters must be used somewhere (eg search or sort). You have not done this in views. If you think to do it in the future - then leave it in the form of comments. Otherwise it's better to delete...